From: YOKOTA Hiroshi <yokota.hgml@gmail.com>
Date: Mon, 23 Nov 2020 22:50:21 +0900
Subject: Use Python 3

"dh_python3" wants "python" for Python 2, and "python3" for Python 3.

Forwarded: not-needed
---
 src/calibre/devices/cli.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/calibre/devices/cli.py b/src/calibre/devices/cli.py
index 722ee0c..ce9bd3a 100755
--- a/src/calibre/devices/cli.py
+++ b/src/calibre/devices/cli.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 
 __license__   = 'GPL v3'
